bundles/org.openhab.binding.http/README.md

```diff
@@ -0,0 +1,120 @@
+| `refresh`         | no       |   30    | Time in seconds between two refresh calls for the channels of this thing. |
+| `timeout`         | no       |  3000   | Timeout for HTTP requests in ms. |
+| `username`        | yes      |    -    | Username for authentication (advanced parameter). |
+| `password`        | yes      |    -    | Password for authentication (advanced parameter). |
+| `authMode`        | no       |  BASIC  | Authentication mode, `BASIC` or `DIGEST` (advanced parameter). |
+| `commandMethod`   | no       |   GET   | Method used for sending commands `GET`, `PUT`, `POST`. |
+| `contentType`     | yes      |    -    | MIME content-type of the command requests. Only used for  `PUT` and `POST`. |
+| `encoding`        | yes      |    -    | Encoding to be used if no encoding is found in responses (advanced parameter). |  
+| `headers`         | yes      |    -    | Additional headers that are sent along with the request. Format is "header=value".| 
+| `ignoreSSLErrors` | no       |  false  | If set to true ignores invalid SSL certificate errors. This is potentially dangerous.|
+
+*Note:* optional "no" means that you have to configure a value unless a default is provided and you are ok with that setting.
+
+## Channels
+
+Each item type has its own channel-type.
+Depending on the channel-type, channels have different configuration options.
+All channel-types (except `image`) have `stateExtension`, `commandExtension`, `stateTransformation`, `commandTransformation` and `mode` parameters.
+The `image` channel-type supports `stateExtension` only.
+
+| parameter               | optional | default     | description |
+|-------------------------|----------|-------------|-------------|
+| `stateExtension`        | yes      |      -      | Appended to the `baseURL` for requesting states. |
+| `commandExtension`      | yes      |      -      | Appended to the `baseURL` for sending commands. If empty, same as `stateExtension`. |
+| `stateTransformation  ` | yes      |      -      | One or more transformation (concatenated with `∩`) applied to received values before updating channel. |
+| `commandTransformation` | yes      |      -      | One or more transformation (concatenated with `∩`) applied to channel value before sending to a remote. |
+| `mode`                  | no       | `READWRITE` | Mode this channel is allowed to operate. `READ` means receive state, `WRITE` means send commands. |
+
+Some channels have additional parameters.
+When concatenating the `baseURL` and `stateExtions` or `commandExtension` the binding checks if a proper URL part separator (`/`, `&` or `?`) is present and adds a `/` if missing.
+
+### `color`
+
+| parameter               | optional | default     | description |
+|-------------------------|----------|-------------|-------------|
+| `onValue`               | yes      |      -      | A special value that represents `ON` |
```

Comment:
If the response (after transformation) is equal to that value, it is interpreted as `ON` (e.g. a response could be `true`, `1` or `eingeschalltet`).

Reply:
Can you put this sentence in the readme? I think it would help the user to understand how this binding is actually working, as you don't provide examples.

When thinking about how this works, I also wouldn't know what to set the transformation parameters to. I think a full example would be really helpful.

Reply:
I added a note for the special values and also examples for the transformation